n2y/plugins/internallinks.py

```diff
@@ -64,22 +63,22 @@ class NotionInternalLink(TextRichText):
 
     def __init__(self, client, notion_data, block=None):
         super().__init__(client, notion_data, block)
-        if block is None or not is_internal_link(
-            self.href, self.block.page.notion_id
-        ):
+        if block is None or not is_internal_link(self.href, self.block.page.notion_id):
             raise UseNextClass
 
     def to_pandoc(self):
         target_id = get_notion_id_from_href(self.href)
         if target_id is None:
-            logger.warning(
+            self.client.logger.warning(
```

Comment:
I think it might make sense to stick with using a module-level logger here, rather than pulling in the logger from the `self.client` property. I might be missing context though-- is there a reason you took this approach instead?

Reply:
Yes, we allow users to pass a logger to the `Client`, so we want all `Block`, `Page`, and `Database` objects to use `self.client.logger` to utilize any logger that may be passed.

Reply:
Ahh, I see. Thanks for that clarification!